addon/components/search-street.hbs

```diff
@@ -0,0 +1,38 @@
+<div class="uk-margin">
+  <label class="uk-form-label" for={{this.guid}}>{{t
+      "ember-gwr.buildingEntrance.fields.street.description.descriptionLong"
+    }}{{if @required " *"}}
+  </label>
+  <PowerSelect
+    @disabled={{@disabled}}
+    @renderInPlace={{true}}
+    @tabIndex={{"0"}}
+    @triggerClass="uk-select"
+    @searchEnabled={{true}}
+    @search={{perform this.search}}
+    @selected={{@street}}
+    @onChange={{@onChange}}
+    @onFocus={{this.handleFocus}}
+    @searchMessage={{t "ember-gwr.components.linkStreet.search"}}
+    @noMatchesMessage={{t "ember-gwr.components.search.noResults"}}
+    @loadingMessage={{t "ember-gwr.components.search.loading"}}
+    id={{this.guid}}
+    class="uk-overflow-hidden"
+    ...attributes
+    as |streetList|
+  >
+    {{streetList.description.descriptionLong}}{{#if
+      streetList.description.descriptionLong
+    }}
+      <small
+        data-test-street-esid
+        class="uk-text-muted uk-text-right uk-text-italic uk-margin-small-left"
+      >{{t "ember-gwr.street.ESID"}}: {{streetList.ESID}}</small>
```

Comment:
The contrast isn't optimal for the hovered upon option. Though integrated, it will probably look different again.

Reply:
I like the added ESID information :ok_hand:

Reply:
Fixed the contrast. I've used `blend-mode` `multiply` to not interfere with UIKit css rules to much and leaf room for custom font settings.